pkg/aks/components/CruAks.vue

```diff
@@ -160,7 +160,9 @@ export default defineComponent({
       }
 
       // track original version on edit to ensure we don't offer k8s downgrades
-      this.originalVersion = this.normanCluster?.aksConfig?.kubernetesVersion;
+      const kubernetesVersion = this.normanCluster?.annotations['management.cattle.io/current-cluster-controllers-version'];
```

Comment:
It looks like this annotation wont be defined while a cluster is provisioning. Could we fall back here to `semver.coerce(this.normanCluster?.aksConfig?.kubernetesVersion)`

Reply:
I'm unsure if this will be correct.. `semver.coerce()` will evaluate to `1.29.0` when provided with 1.29 for a version; 1.29.10 is the actual Kubernetes version.

Reply:
We spoke OOB on this topic. The best we can reliably do with the information available to us at this point is to attempt to infer some Kubernetes version via `semver.coerce()`. It's not 100% correct, but this is at least stable and reliable when compared to the original approach.